src/clusterfuzz/_internal/bot/tasks/utasks/minimize_task.py

```diff
@@ -845,7 +874,7 @@ def check_deadline_exceeded_and_store_partial_minimized_testcase(
     else:
       testcase.set_metadata('minimization_deadline_exceeded_attempts',
                             attempts + 1)
-      tasks.add_task('minimize', testcase_id, job_type)
```

Comment:
Why change this?

Reply:
ooh because I had started to change that and un-did this but left this one out.
i am quite reliant on unit tests apparently
sorry and thank you for catching!